templates/web/common/producers_resources_list.tt.html

```diff
@@ -1,5 +1,5 @@
-<!-- Please include the following code within a list -->
-<!-- Example: <ul> ... [% INCLUDE 'web/common/producers_resources_list.tt.html' %] ... </ul> -->
+[%# Please include the following code within a list
+Example: <ul> ... [% INCLUDE 'web/common/producers_resources_list.tt.html' %] ... </ul> %]
```

Comment:
Does the template engine understand that the first `%]` doesn't close the comment?

Reply:
Not sure..tho it is not giving any kind of error🥲looking at it for clarification👀

Reply:
I think it won't close the comment by first %]